rudder-webapp/SOURCES/rudder-init

```diff
@@ -169,10 +169,10 @@ LDAPInit()
   sed -i "s/^\([^#].*\)%%POLICY_SERVER_HOSTNAME%%/\1${RUDDER_HOSTNAME}/g" $INITPOLICY_PATH
   sed -i "s#^\([^#].*\)%%POLICY_SERVER_ALLOWED_NETWORKS%%#\1$NET#g" $INITPOLICY_PATH
 
-  ${LDAPADD} -f $BOOTSTRAP_PATH &>> $TMP_LOG
+  ${LDAPADD} -f $BOOTSTRAP_PATH & >> $TMP_LOG
```

Comment:
this looks invalid to me
it backgrounds the process, and redirect, rather than redirecting all